lib/database_consistency/helper.rb

```diff
@@ -29,15 +29,17 @@ def connection_config(klass)
       end
     end
 
-    def project_models
+    def project_models(configuration)
```

Comment:
all models helpers eventually have to go through/ delegate to this method. so the check is done only once here

After scanning the source code, I believe passing the configuration as a parameter to this helper method is the best choice. Because Processors all inherit from BaseProcessor which have in their initialize a `attr_reader :configuration` which can be quite complex depending on how users of this gem setup their configuration.
( this instead of doing a plain `Configuration.new` which is already handled in `BaseProcessor#initialize`

then calling
```ruby
next unless configuration.model_enabled?(klass)
```
which delegates to the configuration object which makes sure that a specific model is enabled and is not part of a **disabled database**

this will resolve the issue of the database not existing beforehand @djezzzl